scalableminds/webknossos
# fix dataset color settings

On load, the color of the dataset layer setting is always black. This fix should show the correct layer color instead.

![image](https://user-images.githubusercontent.com/1105056/27028676-ca7c4530-4f65-11e7-900f-b9899a4b0d87.png)


### Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Fix dataset color setting showing wrong color

### Steps to test:
- Start tracing
- Go to Settings --> data set
- See layer color button. Should either be default (white) or whatever value was set during the last tracing session

------
